test/start-stop.node.js

```diff
@@ -180,34 +187,45 @@ types.forEach((type) => {
       it('ipfsd.exec should match exec', () => {
         expect(ipfsd.exec).to.equal(exec)
       })
+    })
 
-      describe('should fail on invalid exec path', function () {
-        this.timeout(20 * 1000)
+    describe('should fail on invalid exec path', function () {
+      this.timeout(20 * 1000)
 
-        before((done) => {
-          const df = IPFSFactory.create(dfConfig)
-          const exec = path.join('invalid', 'exec', 'ipfs')
+      let ipfsd
+      before((done) => {
+        const df = IPFSFactory.create(dfConfig)
+        const exec = path.join('invalid', 'exec', 'ipfs')
 
-          df.spawn({ init: false, start: false, exec: exec }, (err, daemon) => {
-            expect(err).to.not.exist()
-            expect(daemon).to.exist()
+        df.spawn({
+          init: false,
+          start: false,
+          exec: exec
+        }, (err, daemon) => {
+          expect(err).to.not.exist()
+          expect(daemon).to.exist()
 
-            ipfsd = daemon
-            done()
-          })
+          ipfsd = daemon
+          done()
         })
+      })
 
-        it('should fail on init', (done) => {
-          ipfsd.init((err, node) => {
+      after((done) => ipfsd.stop(done))
+
+      it('should fail on init', (done) => {
+        ipfsd.init({ bits: 1024 },
+          (err, node) => {
             expect(err).to.exist()
             expect(node).to.not.exist()
             done()
           })
-        })
       })
     })
 
     describe('start and stop multiple times', () => {
+      // TODO: wont work on windows until we get `/shutdown` implemented in js-ipfs
+      if (isWindows) { return }
```

Comment:
Please use this.skip()

Reply:
Skip doesn't work inside `describes` only `it` blocks.

Reply:
@dryajov can we put that inside every it block then? It signals the viewer that something is not complete.